tests/healthcheck_test.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !skip_healthcheck
```

Comment:
When we want to skip all e2e tests, wouldn't we need a tag to skip these tests too?

Reply:
We'd want to use the same tag (I think `skip_e2e`)

Reply:
Not sure if this situation arises, but let's say mysql and postgres environment variables are set, but sql environment variables aren't. Then we'd have a flag to skip sqlserver_test, but no flag to skip healthcheck_test, which would then fail because healthcheck_test needs connection names from mysql, postgres, and sql. What do you think?

Reply:
So it looks like we don't have a `skip_e2e` in this project, but we definitely should, as the more usual case is to run all tests, or just run unit tests.

So, shortterm `skip_healthcheck` is at least in line with the other build constraints (`skip_postgres`, `skip_mysql`, `skip_sqlserver`).

```
$ grep -r skip_ *
tests/sqlserver_test.go:// +build !skip_sqlserver
tests/mysql_test.go:// +build !skip_mysql
tests/postgres_test.go:// +build !skip_postgres
tests/healthcheck_test.go:// +build !skip_healthcheck
```

To deal with this, we either leave `skip_healthcheck` (and update the instructions in CONTRIBUTING.md please), or we replace all build flags with `skip_e2e` (bigger diff, better result).

Reply:
I think we need to update the build tags such that users can opt out of database-specific tests. @kurtisvg and I talked this over, and came to this idea: let's put the single instance checks into the database-specific e2e tests and introduce an `alldb_test.go` (open to better wording here) where we skip the tests if any of the environment variables are undefined. How does that sound? Cf. https://pkg.go.dev/testing#hdr-Skipping